seacliff/tests/test_rubin_sky_wcs.py

```diff
@@ -0,0 +1,147 @@
+import pickle
+import io
+
+import numpy as np
+from numpy.testing import assert_allclose
+
+import galsim
+import lsst.geom
+
+
+import seacliff
+
+
+def _make_wcs(seed):
+    rng = np.random.RandomState(seed=seed)
+    crpix = lsst.geom.Point2D(
+        rng.randint(low=100, high=200),
+        rng.randint(low=100, high=200),
+    )
+    scale = rng.uniform(low=0.18, high=0.22)
+    theta = rng.uniform(low=0, high=2.0 * np.pi)
+    cd_matrix = np.array(
+        [[scale, 0.0], [0.0, scale]],
+    )
+    costheta = np.cos(theta)
+    sintheta = np.sin(theta)
+    rotmat = np.array(
+        [[costheta, -sintheta], [sintheta, costheta]],
+    )
+    cd_matrix = np.dot(cd_matrix, rotmat)
+    ra = rng.uniform(low=0, high=2.0 * np.pi)
+    dec = np.arcsin(rng.uniform(-1, 1))
+    crval = lsst.geom.SpherePoint(ra, dec, lsst.geom.radians)
+    return lsst.afw.geom.makeSkyWcs(
+        crpix=crpix,
+        crval=crval,
+        cdMatrix=cd_matrix,
+    )
+
+
+def _test_pos(wcs1, wcs2, shift=None):
+    if shift is not None:
+        dx = shift.x
+        dy = shift.y
+    else:
+        dx = 0
+        dy = 0
+    ra1, dec1 = wcs1.xyToradec(
+        np.array([10, 9]),
+        np.array([78, 55]),
+        units=galsim.degrees,
+    )
+    ra2, dec2 = wcs2.xyToradec(
+        np.array([10, 9]) + dx,
+        np.array([78, 55]) + dy,
+        units=galsim.degrees,
+    )
+    assert_allclose(ra1, ra2)
+    assert_allclose(dec1, dec2)
+
+    x1, y1 = wcs1.radecToxy(ra1, dec1, units=galsim.degrees)
+    x2, y2 = wcs2.radecToxy(ra2, dec2, units=galsim.degrees)
+    assert_allclose(x1, x2 - dx)
+    assert_allclose(y1, y2 - dy)
+
+    assert_allclose(x1, np.array([10, 9]))
+    assert_allclose(y1, np.array([78, 55]))
```

Comment:
These test arrays are repeated a few times.  Probably worth making named variables at the top with these values.  (Especially in case some edge case comes along that we'd want to add to the arrays.)